Make get_win_version a static method (#254)
robocorp/robocorp: windows/docs/CHANGELOG.md

# Changelog

## Unreleased

> - Make get_win_version a static method on Desktop

Reviewer: ```suggestion
- Make `get_win_version` a static method of `windows.Desktop` for easier usage.
```
Is more valuable to explain the customer what they get not just what we fixed, so they can asses from a non-technical PoV if the upgrade matters or not for them. Sometimes it helps if we get less technical in the release notes, so even a child can understand the newly brought value.
